src/utils/webrtc/MediaDevicesManager.js

```diff
@@ -126,10 +128,23 @@ MediaDevicesManager.prototype = {
 			value = LOCAL_STORAGE_NULL_DEVICE_ID
 		}
 
-		if (value) {
-			BrowserStorage.setItem(key, value)
+		const numberOfDevices = this.attributes.devices.filter(device => {
```

Comment:
I would prefer all this to be done in the `MediaDevicesPreview` methods using the new `storeInBrowser` parameter.

Reply:
But we also need it with `_updateSelectedDevicesFromGetUserMediaResult` or we make that never store (would also be fine for me)

Reply:
This should not be a problem, as `_updateSelectedDevicesFromGetUserMediaResult` is most likely dead code now since the fix for switching devices in Firefox. But I have not checked, so better keep it just in case ;-) Anyway, not storing there should not be a problem.